src/main/resources/atlassian-plugin.xml

```diff
@@ -8,28 +8,27 @@
     </plugin-info>
 
 
-    					
```

Comment:
oops, my IDE is formatting the files I open to comply with my team's checkstyle
you may ignore this